feature/tpm/tpm.go

```diff
@@ -49,13 +51,22 @@ func init() {
 	}
 }
 
+var verboseTPM = envknob.Bool("TS_DEBUG_TPM")
+
 func info() *tailcfg.TPMInfo {
+	var logf logger.Logf
+	if testenv.InTest() && !verboseTPM {
+		logf = logger.Discard
+	} else {
+		logf = log.New(log.Default().Writer(), "TPM: ", 0).Printf
+	}
```

Comment:
Fine as-written. I'd be tempted to have only one arm

```suggestion
	logf := logger.Discard
	if !testenv.InTest() || verboseTPM {
		logf = log.New(log.Default().Writer(), "TPM: ", 0).Printf
	}
```

but YMMV.